ui/components/InviteMembersModal.tsx

```diff
@@ -228,7 +229,9 @@ const InviteMembersModal = ({
                 onClick={() => {
                   createInviteLink({
                     roundId,
-                  }).then(() => toast.success("Invite link created"));
+                  })
+                  .then(() => toast.success("Invite link created"))
+                  .catch(() => toast.error("Could not created invite link"));
```

Comment:
urql mutations don't reject https://formidable.com/open-source/urql/docs/basics/react-preact/#handling-mutation-errors

Reply:
github just errors when i try to click resolve on this :upside_down_face:

Reply:
On my account, github says it is resolved.